modyn/trainer_server/internal/trainer/pytorch_trainer.py

```diff
@@ -57,7 +57,7 @@
     seed_everything,
 )
 
-AvailableQueues = Enum("AvailableQueues", ["TRAINING", "DOWNSAMPLING"])
+AvailableQueues = Literal["TRAINING", "DOWNSAMPLING"]
```

Comment:
I think it's appropriate to use literals here as the are more lightweight.

Reply:
Fair, probably that's related to flaky linting. In theory, linters / mypy can catch illegal literal values.